frb_codegen/assets/integration_template/shared/REPLACE_ME_RUST_CRATE_DIR/Cargo.toml.template

```diff
@@ -8,3 +8,4 @@ crate-type = ["cdylib", "staticlib"]
 
 [dependencies]
 flutter_rust_bridge = REPLACE_ME_RUST_FRB_DEPENDENCY
+log = "0.4.20"
```

Comment:
```suggestion
log = "0.4.20" # Optional - can be removed if you do not want logging
```

Reply:
I needed to introduce a dependency to `chrono` as well ...
I marked both as
```
log = "0.4.20"  # Optional - can be removed if you do not want logging
chrono = "0.4.23"  # Optional - can be removed if you do not want logging (needed only for the default log output)
```